openide.modules/src/org/openide/modules/AboutMessageOverride.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.openide.modules;
+
+import org.openide.util.lookup.ServiceProvider;
+
+/**
+ * An interface to allow you to change what is displayed in the about window.
+ * Register it with {@link ServiceProvider}
+ * @author astephens
+ */
+public interface AboutMessageOverride {
```

Comment:
I think this may be considered as an API change? If so, it should be accompanied by a version bump and a quick API review process, right @JaroslavTulach?

Reply:
Yes, this looks like an API change. In such case I suggest to follow http://wiki.netbeans.org/APIReviews (to certain extent). Thanks for CCing me, @mdindoffer!